composer/src/lib.rs

```diff
@@ -1014,6 +1100,21 @@ impl ComposeTest {
         Ok(())
     }
 
+    /// stop all the containers part of the network
+    /// returns the last error, if any or Ok
+    pub async fn stop_network_containers(&self) -> Result<(), Error> {
+        let mut result = Ok(());
+        let containers = self.list_network_containers(&self.name).await?;
+        for container in containers {
+            let id = container.id.unwrap_or_default();
```

Comment:
Nit: If we end up using the default value we would see the print message printed with an empty id. Should we just test for the default value and move on to the next container rather than trying to stop it?

Reply:
yeah that's better, I'll do that :+1: